content/guidance/become-a-teacher-in-england.md

```diff
@@ -884,14 +619,11 @@ If you're struggling to find suitable referees, you can:
 
 ### Attend an interview
 
-If your application is successful, you’ll be invited for an interview by
-your training provider.
+If your application is successful, you’ll be invited for an interview by your training provider.
 
-Interviews usually last a day and include an interview and some time
-spent teaching a class of real pupils.
+Interviews usually last a day and include an interview and some time spent teaching a class of real pupils.
```

Comment:
This sounds a touch clunky, I'd be worried if the interview _didn't_ include an interview.

Per
```suggestion
Interviews usually last a day and may include some time spent teaching a class of real pupils.
```